src/app/story/directives/podcast.component.ts

```diff
@@ -33,12 +37,53 @@ export class PodcastComponent implements OnDestroy {
 
   story: StoryModel;
   tabSub: Subscription;
-  itunesExplicitDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesRequirementsDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesExplicitDoc = 'https://support.apple.com/en-us/HT202005';
 
-  OPTIONS = ['Explicit', 'Clean'];
+  OPTIONS = ['', 'Explicit', 'Clean'];
 
-  constructor(tab: TabService) {
-    this.tabSub = tab.model.subscribe((s: StoryModel) => this.story = s);
+  constructor(tab: TabService,
+              private cms: CmsService) {
+    this.tabSub = tab.model.subscribe((s: StoryModel) => this.init(s));
+  }
+
+  init(story) {
+    this.story = story;
+    this.getSeriesOverrides();
+  }
+
+  getSeriesOverrides() {
+    this.cms.account.subscribe(account => {
+      if (this.story.isNew && this.story.parent && this.story.parent.isa('series')) {
+        this.defaultDistributionOverrides(new SeriesModel(account, this.story.parent, false));
+      }
+    });
+  }
+
+  defaultDistributionOverrides(series: SeriesModel) {
+    series.loadRelated('distributions').subscribe(distributions => {
+      this.defaultExplicitToSeries(<DistributionModel[]>distributions);
+    });
+  }
+
+  defaultExplicitToSeries(distributions: DistributionModel[]) {
+    let dist = distributions.find(d => d.kind === 'podcast');
+    dist.loadRelated('versionTemplate').subscribe(() => {
+      dist.loadExternal().subscribe(() => {
+        this.story.loadRelated('versions').subscribe(() => {
+          this.story.versions.forEach(v => {
+            // well crap, new story's version doc is null
```

Comment:
@cavis This is pretty much where I keep getting stuck. I'm this far along, but since the story is new, its versions do not have a doc. (I do intend to add the relation on audio versions to audio version templates and loadRelated, just working with what I've got for now.)